### DISCUSSION TITLE
某安全数据交换系统的漏洞挖掘 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#28] [General]
# 某安全数据交换系统的漏洞挖掘 | 倾旋的博客

本文写于2022年，分享一下挖掘某安全数据交换系统漏洞的过程。

https://payloads.online/archivers/2023/09/18/acc369fd-9310-4351-889c-457b12da9c25

**Comment by yupd** [comment]
大神vmware虚拟机可以分享给我一份吗？学习学习。
